### PR TITLE
Changes need in bgp provider to rescue unsupported features

### PR DESCRIPTION
**Pull request by RichWellum**
Adding rescues where needed.

### CODE REVIEW

File: lib/puppet/provider/cisco_bgp/nxapi.rb [hunk 1]
```diff
@@ -230,7 +232,11 @@ def puts_config
     current = sprintf("\n%30s: %s", 'asn', @bgp_vrf.asnum)
     current.concat(sprintf("\n%30s: %s", 'vrf', @bgp_vrf.vrf))
     BGP_ALL_PROPS.each do |prop|
-      current.concat(sprintf("\n%30s: %s", prop, @bgp_vrf.send(prop)))
+      begin
+          current.concat(sprintf("\n%30s: %s", prop, @bgp_vrf.send(prop)))
+        rescue Cisco::UnsupportedError
+          nil
+        end
```
Comment (robert-w-gries) [inline lib/puppet/provider/cisco_bgp/nxapi.rb]: Indentation is off.  What's strange is that I ran rubocop 0.34.2 against this file, and it's not catching it.

File: lib/puppet/provider/cisco_bgp/nxapi.rb [hunk 2]
```diff
@@ -230,7 +232,11 @@ def puts_config
...
     end
     debug current
   end # puts_config
```
Comment (chrisvanheuveln) [inline lib/puppet/provider/cisco_bgp/nxapi.rb]: :+1: after indentation fix

**Comment by glennmatthews** [comment]
This is no longer needed after cisco/cisco-network-node-utils#82, correct?

**Comment by RichWellum** [comment]
Yes correct - will have a new diff out shortly.

**Pull request closed without merging**